src/data/analyse_de.json

```diff
@@ -0,0 +1,309 @@
    "content": "<strong>Täglich</strong><p>Tagesgenaue Anzahl der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten seit Beginn der Messung.</p><br><strong>Weitere Informationen</strong><p>Für das Beenden der Infektionsketten sind nur die positiven Testergebnisse relevant. Um Missbrauch zu verhindern, müssen positive Testergebnisse verifiziert werden. Dafür gibt es zwei Möglichkeiten: einen QR-Code oder eine teleTAN, die in der Corona-Warn-App eingegeben werden müssen. Den QR-Code erhält die/der Nutzende bei der Probenentnahme für den Test. Mit Hilfe des QR-Codes kann der Test in der Corona-Warn-App registriert werden. Sobald das Ergebnis vorliegt, wird es automatisch abgerufen und auf dem Smartphone angezeigt. Die/der Nutzende kann dann entscheiden, die eigenen Zufallscodes der letzten 14 Tage freizugeben und mögliche Risikokontakte zu warnen. Steht kein QR-Code zur Verfügung oder geht dieser verloren, kann die/der Nutzende die Verifizierungshotline anrufen. Dort wird eine teleTAN zur Verifizierung des positiven Testergebnisses erzeugt. Diese muss durch die/den Nutzenden in die App eingegeben werden.</p><br><strong>Reiter: QR-Codes</strong><strong>Täglich</strong><p>Tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes seit Beginn der Messung.</p><br><strong>Reiter: teleTANs</strong><strong>Täglich</strong><p>Tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten teleTANs.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der in der Corona-Warn-App geteilten und nicht geteilten teleTANs (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten teleTANs.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der in der Corona-Warn-App geteilten und nicht geteilten teleTANs seit Beginn der Messung.</p>"
```

Comment:
I would not use "muss" but rather "kann". Yes, it makes only sense to also use the TAN when you requested one, but you don't have to. 
```suggestion
                "content": "<strong>Täglich</strong><p>Tagesgenaue Anzahl der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der in der Corona-Warn-App geteilten QR-Codes und teleTANs sowie die Summe beider Teilungsmöglichkeiten seit Beginn der Messung.</p><br><strong>Weitere Informationen</strong><p>Für das Beenden der Infektionsketten sind nur die positiven Testergebnisse relevant. Um Missbrauch zu verhindern, müssen positive Testergebnisse verifiziert werden. Dafür gibt es zwei Möglichkeiten: einen QR-Code oder eine teleTAN, die in der Corona-Warn-App eingegeben werden müssen. Den QR-Code erhält die/der Nutzende bei der Probenentnahme für den Test. Mit Hilfe des QR-Codes kann der Test in der Corona-Warn-App registriert werden. Sobald das Ergebnis vorliegt, wird es automatisch abgerufen und auf dem Smartphone angezeigt. Die/der Nutzende kann dann entscheiden, die eigenen Zufallscodes der letzten 14 Tage freizugeben und mögliche Risikokontakte zu warnen. Steht kein QR-Code zur Verfügung oder geht dieser verloren, kann die/der Nutzende die Verifizierungshotline anrufen. Dort wird eine teleTAN zur Verifizierung des positiven Testergebnisses erzeugt. Diese kann durch die/den Nutzenden in die App eingegeben werden.</p><br><strong>Reiter: QR-Codes</strong><strong>Täglich</strong><p>Tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der in der Corona-Warn-App geteilten und nicht geteilten QR-Codes seit Beginn der Messung.</p><br><strong>Reiter: teleTANs</strong><strong>Täglich</strong><p>Tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten teleTANs.</p><strong>Wöchentlich</strong><p>Wöchentliche Anzahl der in der Corona-Warn-App geteilten und nicht geteilten teleTANs (nach Kalenderwoche (KW)).</p><strong>7-Tage-Mittelwert</strong><p>Tagesgenauer Mittelwert der letzten sieben Tage und tagesgenaue Anzahl der in der Corona-Warn-App geteilten und nicht geteilten teleTANs.</p><strong>Kumuliert</strong><p>Tagesgenaue kumulative Summe der in der Corona-Warn-App geteilten und nicht geteilten teleTANs seit Beginn der Messung.</p>"
```